IPython/core/magics/execution.py

```diff
@@ -965,8 +965,10 @@ def timeit(self, line='', cell=None, local_ns=None):
         -n<N>: execute the given statement <N> times in a loop. If this value
         is not given, a fitting value is chosen.
 
-        -r<R>: repeat the loop iteration <R> times and take the best result.
-        Default: 3
+        -r<R>: number of runs<R>, each consisting of <N> loops, and take the
+        best result. If <N> is not provided, <R> is determined so as to get
```

Comment:
It's `<N>` that changes to get sufficient accuracy, not `<R>` - and that note should be in the description of the `-n` flag.

Also, you've got rid of the word *repeat*, which is what the R stands for.

Reply:
@takluyver.. when I was reading the documentation in the code.. I was a bit confused because when you run something like `%timeit -r3 100**100`, it will print out something like 

`12.4 ns ± 0.187 ns per loop (mean ± std. dev. of 3 runs, 100000000 loops each)`. 

So it seemed that `run` was a better description of what `-r` stands for. Otherwise.. would it be better to printout something like

`12.4 ns ± 0.187 ns per loop (mean ± std. dev. of 3 repeats, 100000000 loops each)`.

In that case I can change `runs` to `repeats`: https://github.com/alphaCTzo7G/ipython/blob/b26f764c594aa83d2bb1f6efcd6c94045f3f5fe0/IPython/core/magics/execution.py#L97?

If you guys think, its better to keep the word `repeat` in the documentation and `run` in the string printed out in `%timeit`.. I can just keep it as is and reword  the commit as well